src/shell/components/InAppAnnouncement/index.tsx

```diff
@@ -0,0 +1,155 @@
+import { useState, useMemo } from "react";
+import { ThemeProvider } from "@mui/material";
+import { theme } from "@zesty-io/material";
+import moment from "moment";
+import { useLocalStorage } from "react-use";
+import {
+  Dialog,
+  DialogContent,
+  DialogActions,
+  Typography,
+  Stack,
+  Button,
+  Box,
+} from "@mui/material";
+import OpenInNewRoundedIcon from "@mui/icons-material/OpenInNewRounded";
+import ScheduledRoundedIcon from "@mui/icons-material/ScheduleRounded";
+import PlayArrowRoundedIcon from "@mui/icons-material/PlayArrowRounded";
+
+import { useGetAnnouncementsQuery } from "../../services/marketing";
+
+export const InAppAnnouncement = () => {
+  const [activeAnnouncementIndex, setActiveAnnouncementIndex] = useState(0);
+  const [readAnnouncements, setReadAnnouncements] = useLocalStorage(
```

Comment:
Local storage isolates data by origin. This is causing the pop-up to show on ever instance a user visits. We want to avoid that and only show the announcement once per user. 2 alternatives I can think of are using a cookie or storing it to the [users account](https://accounts-api.zesty.org/#f81edf43-a115-4a2e-b113-4be60205ffc9)

Reply:
https://github.com/zesty-io/accounts-api/issues/244